src/components/organisms/UiMultipleAnswer/UiMultipleAnswer.vue

```diff
@@ -175,7 +175,7 @@ const itemsToRender = computed(() => (props.items.map((item) => {
     ...item,
     inputAttrs: {
       name: props.name,
-      ...item.inputAttrs,
+      ...((typeof item.inputAttrs === 'object') ? item.inputAttrs : {}),
```

Comment:
can't we use that? 
```...item.inputAttrs || {}```

Reply:
No, ts screams at me:
![image](https://github.com/infermedica/component-library/assets/37744371/8bc505c3-cfe1-47ab-a4da-5669dc5cbbba)